models/public/gmcnn-tf/gmcnn-tf.md

```diff
@@ -0,0 +1,146 @@
+
+### Steps to Reproduce Conversion to Frozen Graph
+
+1. Clone the original repository
+```sh
+git clone https://github.com/shepnerd/inpainting_gmcnn.git
+cd inpainting_gmcnn/tensorflow
+```
+2. Checkout the commit that the conversion was tested on:
+```sh
+git checkout ba7f710
+```
+3. Install the [original dependencies](https://github.com/shepnerd/inpainting_gmcnn#prerequisites).
+(TensoFlow\* version used - 1.14.0, CPU).
+4. Download the [pretrained weights](https://drive.google.com/file/d/1aakVS0CPML_Qg-PuXGE1Xaql96hNEKOU/view?usp=sharing)
+5. Run sample [conversion script](./freeze_model.py):
+```sh
+python3 freeze_model.py --ckpt_dir path/to/downloaded_weights --save_dir path/to/save_directory
+```
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Image Inpainting                          |
+| GFlops                          | 371.2                                     |
+| MParams                         |                                           |
+| Source framework                | TensorFlow\*                              |
+
+## Accuracy
+
+Accuracy metrics are obtained on 2000 image subset of VOC2012 dataset. Images were cropped to input size
+and disguised at random positions with pre-generated free-form masks.
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| PSNR   | 33.47Db        | 33.47Db         |
+
+## Performance
+
+## Input
+
+### Original Model
+
+1. Image, name: `Placeholder`, shape: [1x512x680x3], format: [BxHxWxC]
+  where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: RGB.
+
+2. Mask, name: `Placeholder_1`, shape: [1x512x680x1], format: [BxHxWxC]
+  where:
+
+    - B - batch size
+    - H - mask height
+    - W - mask width
+    - C - number of channels
+
+### Converted Model
+
+1. Image, name: `Placeholder`, shape: [1x3x512x680], format: [BxCxHxW]
+  where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
+
+2. Mask, name: `Placeholder_1`, shape: [1x1x512x680], format: [BxCxHxW]
+  where:
+
+    - B - batch size
+    - C - number of channels
+    - H - mask height
+    - W - mask width
+
+## Output
+
+### Original Model
+
+Restored image, name `Cast`, shape: [1x512x680x3], format: [BxHxWxC]
+  where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: RGB.
+
+### Converted Model
+
+Restored image, name: `Cast`, shape: [1x3x512x680], format: [BxCxHxW]
+  where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
```

Comment:
Something isn't right here. In the documentation, the color order is flipped in the converted model, but there's no `--reverse_input_channels` flag in the config file.

Reply:
There was a bug in documentation, fixed